tests/Operation/ListCollectionsFunctionalTest.php

```diff
@@ -34,6 +34,33 @@ public function testListCollectionsForNewlyCreatedDatabase()
         }
     }
 
+    public function testOffsetGet()
+    {
+        if (version_compare($this->getServerVersion(), '3.4.0', '<')) {
+            $this->markTestSkipped('idIndex and info are not supported');
+        }
+
+        $server = $this->getPrimaryServer();
+
+        $operation = new DropDatabase($this->getDatabaseName());
+        $operation->execute($server);
+
+        $insertOne = new InsertOne($this->getDatabaseName(), $this->getCollectionName(), ['x' => 1]);
+        $writeResult = $insertOne->execute($server);
+        $this->assertEquals(1, $writeResult->getInsertedCount());
+
+        $operation = new ListCollections($this->getDatabaseName(), ['filter' => ['name' => $this->getCollectionName()]]);
+        $collections = $operation->execute($server);
+
+        $this->assertInstanceOf('MongoDB\Model\CollectionInfoIterator', $collections);
+
+        foreach ($collections as $collection) {
+            $this->assertInstanceOf('MongoDB\Model\CollectionInfo', $collection);
+            $this->assertArrayHasKey('readOnly', $collection['info']);
+            $this->assertEquals($collection['idIndex'], ['v' => 2, 'key' => ['_id' => 1], 'name' => '_id_', 'ns' => $this->getNamespace()]);
```

Comment:
The expected value comes before the actual value in [`assertEquals()`](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertEquals) (and other assertions). This makes no difference for a passing test but becomes important when PHPUnit renders an error message for a test failure.